### ISSUE TITLE
[BUG] Cannot upload new documents anymore

### ISSUE THREAD

**Issue by siancu** [paperless-ngx/paperless-ngx#2164]
### Description

I am running paperless-ngx 1.10.2 in docker on a Synology NAS. All of a sudden, I cannot upload any new documents anymore.

I have watchtower running and the last version where documents uploading worked was 1.9.2 (I think). 

### Steps to reproduce

1. Upload new documents either by drag-and-drop on the Dashboard or by pressing the browse files button.
2. The documents stay in "Upload complete, waiting..." state for long time (I waited something like 5 minutes, before it was taking seconds)
3. If I refresh the page, the documents are "lost". They are no longer in Processing nor in the Inbox. The count of total documents stays the same.

### Webserver logs

```bash
There are no errors whatsoever in the logs. The last things I see are as follows:

[2022-12-11 12:03:02,870] [INFO] [celery.app.trace] Task documents.tasks.train_classifier[fdc062a0-3a3f-4058-8301-0bd81e01b317] succeeded in 25.466245260089636s: None

[2022-12-11 12:03:03,467] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[5dd45ed7-a128-4583-8c8b-77f6f81dd661] succeeded in 0.2429492101073265s: 'No new documents were added.'

[2022-12-11 12:03:03,477] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[08a8fd9f-f8a2-49ce-99e2-737fc2da9304] received

[2022-12-11 12:03:04,034] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[6ef2e619-3568-4b22-89d6-f66e01fca57f] succeeded in 0.24232304003089666s: 'No new documents were added.'

[2022-12-11 12:03:04,044] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[fdec8642-9942-41e4-9c1f-d211edcb47e7] received

[2022-12-11 12:03:04,419] [INFO] [celery.worker.strategy] Task documents.tasks.train_classifier[57389e44-ee8a-46c6-8475-d95e4341eff4] received

[2022-12-11 12:03:04,711] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[2a177f76-4e6d-4480-8dc2-83fec8280dc2] succeeded in 0.32266412395983934s: 'No new documents were added.'

[2022-12-11 12:03:05,238] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[34c8f744-a630-44b0-9bea-d0bc45e0c36e] received

[2022-12-11 12:03:05,318] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[1d9d817e-98ca-4d6b-8f30-d87908a7d21d] succeeded in 0.28442465607076883s: 'No new documents were added.'

[2022-12-11 12:03:05,953] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[08a8fd9f-f8a2-49ce-99e2-737fc2da9304] succeeded in 0.3033050619997084s: 'No new documents were added.'

[2022-12-11 12:03:05,968] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[096e4578-d7a1-4ab1-86de-1adbd0288440] received

[2022-12-11 12:03:06,594] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[7261665e-74fe-4a67-a628-3afcc8783906] received

[2022-12-11 12:03:06,861] [INFO] [celery.app.trace] Task paperless_mail.tasks.process_mail_accounts[fdec8642-9942-41e4-9c1f-d211edcb47e7] succeeded in 0.5830680876970291s: 'No new documents were added.'

[2022-12-11 12:03:07,693] [INFO] [celery.worker.strategy] Task paperless_mail.tasks.process_mail_accounts[42f498e2-6810-4eba-8c2b-1e06a22a611c] received
```

### Browser logs

```bash
Also the browser doesn't show any errors. I am trying to upload two documents so in the Network tab of the browser I see two POST requests and the response for both is "OK":
```

### Paperless-ngx version

1.10.2

### Host OS

Synology NAS - x86_64

### Installation method

Docker - official image

### Browser

Firefox

**Comment by siancu** [comment]
I just noticed in "File Tasks" that all my tries to add the documents are in the "Queued" state and they seem to be stuck in this state.

**Comment by siancu** [comment]
I restarted the whole compose stack and now it seems to be working again. Don't know exactly what the cause was, but I'm going to close this ticket now.

**Comment by d-d** [comment]
Same happened to me. Synology+watchtower.  redis automatically updated by watchtower to 7.0.7.
Uploaded documents would get stuck in "queued" mode, but were never consumed.

I followed the suggestion in https://github.com/paperless-ngx/paperless-ngx/issues/1003#issuecomment-1135104259
and I stopped the redis container, deleted the dump.rdb redis data from the redis volume, and then restarted the redis container. 

That solved it. Documents are now consumed again.
